src/main/java/github/scarsz/discordsrv/modules/requirelink/RequireLinkModule.java

```diff
@@ -192,10 +206,15 @@ public void noticePlayerUnlink(Player player) {
         if (checkWhitelist()) {
             boolean whitelisted = Bukkit.getServer().getWhitelistedPlayers().stream().map(OfflinePlayer::getUniqueId).anyMatch(u -> u.equals(player.getUniqueId()));
             if (whitelisted) {
-                DiscordSRV.debug("Player " + player.getName() + " is bypassing link requirement, player is whitelisted");
+                DiscordSRV.debug("Player " + player.getName() + " is bypassing link requirement beacuse Only check banned players is enabled");
```

Comment:
Changed the debug message in the wrong spot? Looks like this was suppose to be on line 215 instead

Reply:
Oops. You are right. Thanks!